client/state/data-layer/wpcom/sites/rewind/index.js

```diff
@@ -27,17 +33,76 @@ const fetchRewindState = action =>
 		action
 	);
 
+/**
+ * Re-requests the activity log entries for all
+ * previously-sent queries in application state
+ *
+ * This function is highly coupled to the details
+ * of Activity Log and Rewind state!
+ *
+ * It will not grab events we haven't already
+ * gotten. Instead it will reissue the same reqeusts
+ * as previously made so that we get updates on
+ * things like the `isDiscarded` status of events.
+ *
+ * @param {Number} siteId the site being queried
+ * @param {Object} rewind Rewind state for the site
+ * @returns {function} thunk action maybe dispatching requests
+ */
+const refreshActivityLogAfterRewind = ( siteId, rewind ) => ( dispatch, getState ) => {
+	if ( ! rewind || rewind.status === 'running' ) {
+		return;
+	}
+
+	const state = getState();
+	const prev = state.rewind[ siteId ];
+
+	if ( ! prev || ! ( prev.rewind && prev.rewind.status === 'running' ) ) {
+		return;
+	}
+
+	const logItems = state.activityLog.logItems[ siteId ];
+	if ( ! logItems ) {
+		return;
+	}
+
+	const queries = logItems.data.queries;
+	Object.keys( queries ).forEach( query =>
```

Comment:
This `queries` const is redundant since it's never used again and doesn't even introduce more readability.

Reply:
inlined in ee158ae948